repos/system_upgrade/el8toel9/actors/opensslconfigscanner/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import readconf
+from leapp.models import OpenSslConfig
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class OpenSslConfigScanner(Actor):
+    """
+    Read an OpenSSL configuration file for further analysis.
+    """
+
+    name = 'open_ssl_config_scanner'
+    consumes = ()
+    produces = (OpenSslConfig)
```

Comment:
sorry, just realized that now. In python `(OpenSslConfig)` is transformed to `OpenSslConfig` (no tuple) when just one item is preset. To preserve the tuple, it's needed to use `(OpenSslConfig, )` (comma in the end)
```suggestion
    produces = (OpenSslConfig,)
```

It's not blocker for the merge, so keeping on you, but it would be better

Reply:
Added. Missed this one even though I did correctly in all the other places.